src/components/Compare/Compare.style.tsx

```diff
@@ -272,6 +273,25 @@ export const Rank = styled.span`
   margin-right: 0.75rem;
 `;
 
+export const LocationCellWrapper = styled.div`
+  display: flex;
+  min-width: 0;
+`;
+
+export const LocationRankWrapper = styled.div`
+  flex: 0 0 auto;
+`;
+
+export const LocationNameWrapper = styled.div`
+  flex: 1 1 auto;
+  white-space: normal;
+  text-overflow: initial;
+`;
+
+export const LocationRankLink = styled(Link)`
```

Comment:
I'd just call this `StyledLink` since its wrapping the whole row and not just the rank